ha/core/controllers/pcs/node_controller.py

```diff
@@ -188,8 +188,27 @@ def start(self, nodeid: str) -> dict:
             _output, _err, _rc = self._execute.run_cmd(const.PCS_NODE_START.replace("<node>", nodeid), check_error=False)
             if _rc != 0:
                 raise ClusterManagerError(f"Failed to start node {nodeid}")
+
+            Log.info(f'Node: {nodeid} started successfully. Now, waiting for \
```

Comment:
I guess this changes is not related to this ticket, please check.

Reply:
No this change it got some merge problem

Can you check now patch is updated properly